Vagrantfile

```diff
@@ -119,6 +119,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     vb.memory = vm_memory
     vb.cpus = vm_num_cpus
   end
+  config.vm.provider "parallels" do |vb, override|
+    config.vm.box = "bento/ubuntu-20.04"
```

Comment:
We use 18.04 everywhere else. Does that not work here?

Reply:
Actually, `hashicorp/bionic64` does not support parallels, so I went with most popular parallels supported ubuntu box. I can try to find a ubuntu-18 box for this.